index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Welcome!</title>
+  </head>
+  <body>
+    <time id="time" class="output-field"></time>
+    <h1 class="output-field">
+      <span id="greeting">Good day,</span>
+      <span id="name" contenteditable="true">Enter you name here...</span>
+    </h1>
+    <!-- <h2 class="output-field">Погода в Минске</h2> -->
+
+    <!-- <h2 id="addtask" contenteditable="true">Enter new task</h2> -->
+    <input id="addtask" placeholder="Enter new task here and press Enter" />
+    <div id="tasklist">
+      <ul class="listoftasks"></ul>
```

Comment:
Please, be more attentive with naming. I recommend use google style guide https://google.github.io/styleguide/htmlcssguide.html

Reply:
done